fluent-bit-firehose.go

```diff
@@ -90,6 +90,11 @@ func newFirehoseOutput(ctx unsafe.Pointer, pluginID int) (*firehose.OutputPlugin
 func FLBPluginInit(ctx unsafe.Pointer) int {
 	plugins.SetupLogger()
 
+	logrus.Info("A new higher performance Firehose plugin has been released; " +
```

Comment:
See my comment on the cloudwatch one: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/127#discussion_r527487281

I purposefully made this message Info, whereas for CloudWatch it is debug. That is because continue to use the golang CloudWatch plugin is valid... it has some very useful features which the new core one lacks.

However, for Firehose users, there is mostly not a good reason for them to continue to use this plugin. I think essentially all of them could migrate, the `replace_dots` option which is unique to this plugin is probably not widely used. Therefore, the migration notice is an Info statement on startup so that most users will see it.